src/update/PluginVersion.cpp

```diff
@@ -3,7 +3,7 @@
 
 namespace UKControllerPlugin {
     namespace Plugin {
-        const char * const PluginVersion::version = "1.3.0";
+        const char * const PluginVersion::version = "1.4.1";

```

Comment:
Is this still current?

Reply:
Yeah - this is just a mashup with master, master is on 1.4.x :)